docs/src/basics/solve.md

```diff
@@ -8,14 +8,19 @@ solve(prob::SciMLBase.NonlinearProblem, args...; kwargs...)
 
   - `alias_u0::Bool`: Whether to alias the initial condition or use a copy.
     Defaults to `false`.
-  - `internal_norm::Function`: The norm used by the solver. Default depends on algorithm
+  - `internalnorm::Function`: The norm used by the solver. Default depends on algorithm
     choice.
 
 ## Iteration Controls
 
   - `maxiters::Int`: The maximum number of iterations to perform. Defaults to `1000`.
-  - `abstol::Number`: The absolute tolerance. Defaults to `real(oneunit(T)) * (eps(real(one(T))))^(4 // 5)`.
-  - `reltol::Number`: The relative tolerance. Defaults to `real(oneunit(T)) * (eps(real(one(T))))^(4 // 5)`.
+  - `maxtime`: The maximum time for solving the nonlinear system of equations. Defaults to
+    `nothing` which means no time limit. Note that setting a time limit does have a small
+    overhead.
+  - `abstol::Number`: The absolute tolerance. Defaults to
+    `real(oneunit(T)) * (eps(real(one(T))))^(4 // 5)`.
+  - `reltol::Number`: The relative tolerance. Defaults to
```

Comment:
These can be vectors matching the size as well?

Reply:
No that is not implemented.

Reply:
We should make an issue on this then. It just needs to use broadcast?